pypeclub/OpenPype
# Global: adding studio name/code to anatomy template formatting data

## Brief description
Adding some enhancement for slate generator addon

## Description
Slate generator addon needs some handling to be done on openpype side

## Additional info
- studio name and code is added to all anatomy formatting data

# Dependencies
avalon-core: https://github.com/pypeclub/avalon-core/pull/417


## Testing notes:
1. add to anatomy template workfile to folder and file `{studio[code]}`
2. open any task and see that studio code was used in path creation
3. open workfile tool and create new - look to example name that studio code was used.
4. if you add studio code/name in burnins then try it and test.
